install.sh

```diff
@@ -573,11 +573,11 @@ EOABORT
 This installation may not succeed.
 After installation, you will encounter build failures with some formulae.
 Please create pull requests instead of asking for help on Homebrew\'s GitHub,
-Twitter or IRC. You are responsible for resolving any issues you experience
-while you are running this ${what}.
+Twitter or any other official channels. You are responsible for resolving any
+issues you experience while you are running this ${what}.
 EOS
     )
-"
+" | tr -d "\\"
```

Comment:
Just remove the backslash above.

```
Homebrew\'s GitHub --> Homebrew's GitHub
```

Reply:
I did that originally, but it turns out a bare single quote in that context triggers a strange bug further down the file.
```
./install.sh: line 965: syntax error near unexpected token `('
./install.sh: line 965: `    echo 'eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"' >> ${shell_profile}'
```

Reply:
We can use `cat <<"EOS"` to avoid substitution for escape characters.

Reply:
> We can use `cat <<"EOS"` to avoid substitution for escape characters.

Oh, I'm wrong. We need to expand ${what} in the message.

> but it turns out a bare single quote in that context triggers a strange bug further down the file.
>
> ```
> ./install.sh: line 965: syntax error near unexpected token `('
> ./install.sh: line 965: `    echo 'eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"' >> ${shell_profile}'
> ```

I cannot reproduce this with Bash 5 and the error message looks doesn't make sense. If the issue still arises (probably for other versions of Bash?), it is fine for me to use `tr -d`.

Reply:
It affects bash 3.2, which is what macOS still ships with.